### PR TITLE
column lineage docs

### PR DESCRIPTION
**Pull request by pawel-big-lebowski**
Signed-off-by: Pawel Leszczynski <leszczynski.pawel@gmail.com>

### CODE REVIEW

File: docs/integrations/spark/spark.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 3
+sidebar_position: 1
```
Comment (mobuchowski) [inline docs/integrations/spark/spark.md:2]: 😉

File: docs/integrations/spark/spark.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ spark-submit --conf spark.driver.extraJavaOptions=-javaagent:<marquez-spark-jar-
 
 ### From Airflow
 The same parameters passed to `spark-submit` can be supplied from Airflow and other schedulers. If
-using the [marquez-airflow](airflow.md) integration, each task in the DAG has its own Run id
+using the [marquez-airflow](../airflow.md) integration, each task in the DAG has its own Run id
```
Comment (mobuchowski) [inline docs/integrations/spark/spark.md:135]: marquez-airflow -> openlineage-airflow

Comment (mobuchowski) [reply docs/integrations/spark/spark.md:135]: And fix all old marquez references 🙂 

Might be done in separate PR though.

File: docs/integrations/spark/spark_column_lineage.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 1
+---
+
+# Column Level Lineage
```
Comment (mobuchowski) [inline docs/integrations/spark/spark_column_lineage.md:5]: Maybe this should be two docs: one about all-around column level lineage and how OpenLineage understands that regardless of integration, and second, more technical, which focuses on Spark implementation

**Review by mobuchowski — CHANGES_REQUESTED** [review]

File: docs/integrations/spark/spark.md (hunk 1 above)
Comment (pawel-big-lebowski) [reply docs/integrations/spark/spark.md:2]: This jumped deeper into category tree ;-)

File: docs/integrations/spark/spark_column_lineage.md (hunk 3 above)
Comment (pawel-big-lebowski) [reply docs/integrations/spark/spark_column_lineage.md:5]: Yeah, had similar idea but thought it's better to have all in single place up until column level lineage gets integrated elsewhere.

File: docs/integrations/spark/spark.md (hunk 2 above)
Comment (pawel-big-lebowski) [reply docs/integrations/spark/spark.md:135]: I was planning another PR to refurbish whole `spark.md`

**Review by mobuchowski — APPROVED** [review]

**Pull request merged**